specification/subscription/resource-manager/Microsoft.Subscription/preview/2023-05-01-preview/subscriptions.json

```diff
@@ -0,0 +1,300 @@
+        "Allowed",
+        "NotAllowed"
+      ]
+    },
+    "SubscriptionsIntoTenant": {
+      "description": "Blocks the entering of subscriptions into user's tenant.",
+      "type": "string",
+      "x-ms-enum": {
+        "name": "TenantPolicyPermission",
+        "modelAsString": true
+      },
+      "enum": [
+        "Allowed",
+        "NotAllowed"
+      ]
+    },
+    "PutTenantPolicyRequestProperties": {
+      "description": "Put tenant policy request properties.",
+      "type": "object",
+      "properties": {
+        "subscriptionsLeavingTenant": {
+          "$ref": "#/definitions/SubscriptionsLeavingTenant"
+        },
+        "subscriptionsIntoTenant": {
+          "$ref": "#/definitions/subscriptionsIntoTenant"
+        },
+        "exemptedPrincipals": {
+          "description": "List of user objectIds that are exempted from the set subscription tenant policies for the user's tenant.",
+          "type": "array",
+          "items": {
+            "format": "uuid",
+            "type": "string",
+            "example": "00000000-0000-0000-0000-000000000000"
+          }
+        }
+      }
+    },
+    "GetTenantPolicyResponse": {
+      "description": "Tenant policy Information.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Policy Id.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "name": {
+          "description": "Policy name.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "type": {
+          "description": "Resource type.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "properties": {
+          "$ref": "#/definitions/TenantPolicy",
+          "description": "Tenant policy properties."
+        },
+        "systemData": {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData",
+          "readOnly": true
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "GetTenantPolicyListResponse": {
+      "description": "Tenant policy information list.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The list of tenant policies.",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/GetTenantPolicyResponse"
+          }
+        },
+        "nextLink": {
+          "description": "The link (url) to the next page of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "TenantPolicy": {
+      "description": "Tenant policy.",
+      "type": "object",
+      "properties": {
+        "policyId": {
+          "description": "Policy Id.",
+          "type": "string",
+          "readOnly": true
+        },
+        "subscriptionsLeavingTenant": {
+          "$ref": "#/definitions/SubscriptionsLeavingTenant"
+        },
+        "subscriptionsIntoTenant": {
```

Comment:
>subscriptionsIntoTenant [](http://example.com/codeflow?start=9&length=23)

circular reference. Issue could be casing. This is a must fix

Reply:
> > subscriptionsIntoTenant
> 
> circular reference. Issue could be casing. This is a must fix

Fixed